compiler/rustc_target/src/spec/targets/aarch64_unknown_fuchsia.rs

```diff
@@ -1,6 +1,24 @@
-use crate::spec::{SanitizerSet, StackProbeType, Target, TargetOptions, base};
+use crate::spec::{Cc, LinkerFlavor, Lld, SanitizerSet, StackProbeType, Target, base};
 
 pub(crate) fn target() -> Target {
+    let mut base = base::fuchsia::opts();
```

Comment:
Should we also set `cpu` just like we do for the other targets?

Reply:
done